airbyte-integrations/connectors/source-klaviyo/source_klaviyo/schemas.py

```diff
@@ -0,0 +1,104 @@
+#
+# MIT License
+#
+# Copyright (c) 2020 Airbyte
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+
+
+from datetime import datetime
+from typing import Any, List, MutableMapping, Optional
+
+from pydantic import BaseModel, Extra
+
+
+class BaseSchemaModel(BaseModel):
+    class Config:
+        extra = Extra.allow
+
+        @staticmethod
+        def schema_extra(schema: MutableMapping[str, Any], model) -> None:
+            for prop in schema.get("properties", {}).values():
+                prop.pop("title", None)
+
+    object: str
+
+
+class PersonList(BaseModel):
```

Comment:
neat. Having gone through this, do you have an opinion on whether this is better than JSON-based schemas?

Reply:
I would say this is much more control over the human mistake and data validation, also it easy to catch mistake with your eyes... still experimenting with generation these model from OpenAPI3

Reply:
generating from OpenAPI3 would be amazing! (but not blocking for this PR 😜)